### ISSUE TITLE
implement "youtube-comment-blacklist"

### ISSUE THREAD

**Issue by super-lurker** [FreeTubeApp/FreeTube#1540]
https://github.com/NatoBoram/youtube-comment-blacklist

like the name suggest, its a userscript that block annoying comments

**Comment by GilgusMaximus** [comment]
I do not understand that, 
Annoying comments are a super subjective view and this might lead to all comments being filtered.

What would be the usecase for this?

**Comment by super-lurker** [comment]
Annoying comments is subjective , but you can customize it, its an option you use it or you don't

**Comment by jasonhenriquez** [comment]
My two cents: I think custom keyword filtering of videos, comments, etc. is a fine idea (similar to what you can do on Twitter), but any preset lists readily available in the app should be on topics like profanity / violence / common triggers. This tool is far too opinionated even if customizable IMO.

**Comment by super-lurker** [comment]
_This tool is far too opinionated even if customizable IMO_

thats the point of having this kind of tool, and ill repeat it again its an option, you can opt out of it ( depending on how they implement it)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> What would be the usecase for this?


Im also wondering what the justification is for implementing this. You still didnt explain this.

**Comment by super-lurker** [comment]
> > What would be the usecase for this?
> 
> Im also wondering what the justification is for implementing this. You still didnt explain it.


to black list youtube comments, I don't know how else to explain it better than that.

but if you need examples comments like:

first

second

me: """ """"
them: """"" """""

click this link to """"" """""

these are the comments I block, they legitimately add zero substance and most of the time they bury actual comments,

**Comment by GilgusMaximus** [comment]
@jasonhenriquez yes, good idea. Especially with additions to parental controls, this might be helpful.

The question remaining is whether these comments should be removed completely or whether they should show something like 'Removed by filter' or similar.
Because it might happen that one loads comments and nothing appears because everything has been filtered.
